src/components/Greeting.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import PropTypes from 'prop-types';
+
+const GET_MESSAGES_REQUEST = 'GET_MESSAGES_REQUEST';
+const GET_MESSAGES_SUCCESS = 'GET_MESSAGES_SUCCESS';
+
+export function getMessagesSuccess(json) {
+  return {
+    type: GET_MESSAGES_SUCCESS,
+    json,
+  };
+}
+
+function getMessages() {
+  return async (dispatch) => {
+    dispatch({ type: GET_MESSAGES_REQUEST });
+    try {
+      const response = await fetch('http://localhost:5000/api/v1/random_message');
+      const json = await response.json();
+      return dispatch(getMessagesSuccess(json));
+    } catch (error) {
+      return console.log(error);
```

Comment:
[Optional]
- [ ]  Instead of outputting an error to the console, you should try to show a message (on HTML) to the user. Console outputs are not user-friendly.

Reply:
You are right, I totally forgot to add those files for gitignore. Thanks.